Corona-Warn-App/src/debug/AndroidManifest.xml

```diff
@@ -15,4 +15,8 @@
         android:name="android.permission.CHANGE_CONFIGURATION"
         tools:ignore="ProtectedPermissions" />
 
+    <application
+        android:requestLegacyExternalStorage="true"
```

Comment:
Why is this needed? Is it visible to the user as permission in the playstore?

Reply:
It is needed to avoid new storage api restrictions in Android 29, So I can save screenshots on `sdcard/..`. it is the path where firebase can pull screenshots from according to their docs. [or the only path that really worked for me  :)]
https://cloud.google.com/sdk/gcloud/reference/firebase/test/android/run#--directories-to-pull.
AND it is not affecting end user at all. it is just debug builds as you can see the link of the file ⬆️  `/debug/AndroidManifest.xml`